### DISCUSSION TITLE
EddieHub badge for our GitHub profile and GitHub Organisation

### DISCUSSION THREAD

**Discussion by vinzvinci** [EddieHubCommunity/support#758] [Ideas]
### Should we put this in our GitHub profile for representing our guild specially when there's an event?

<a href="https://github.com/EddieJaoudeCommunity"><img src="https://img.shields.io/static/v1?label=Guild&message=EddieHub&color=blueviolet&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAChVBMVEUPCwukzr0WFhWBhnYzMzAAAAAPEhUCAgc2LS4lJSQhIB8ODw4oKSgcGhsbHRsXFhUZGRk8My4hISD///+6nIyVcV5DODNsWE0aMFxASVdxW1EhO29JVWkLLnQTIUMAPJ5IPjISEhApMTYbAgBLT1FtZWQKEhFZAgIyDw9tAgKJDAtOBgcAAAAAAAAUFRMhIiEODQ0HBwcBAAEBAAAREBElJiQxMi4xMiwvMSswMiwnJyMeHxsYGRYVFRMbGhkKCAgfHh4eHxwsLSg2ODElJiIpKSc1NDQnJCMgIB4cHRoaGxgpKiktLS8VFBMSEhAgICAFAAAGBgUNDg5dSDyqh3MVFBQqKyoGCAqfgXJ5Z12tineEa1s7NjIAAAAAAAI2LyxhVUxzZVt8aF2cemmwh3KHaFg7MCuDYU+OfXOMf3eOf3VPRk50WlCBY1QrQmsHIlg8QVkmOmcLJlwpPWNGTEMwSHgOLGheaoQ+SGshNWgnQHJYW10AAJ8bNWtoa3STlJOKjZFHS1k6OEIAAB89QEY7OjonJyYmJiYxMTMgIB8FAwA4LS1APz8wMC4ZGBdEQ0M2ODciJiVFAABtCgtiJCRFJyYACAhPKytSMzRIJiYSAABYAAGGBASHBQRpAgFmBwZoCAhkBwc0BAY5PDU6PDY4OjMzNS8wMSsrLSgdHxsjIh0tKSMoJiE3MixiUkdoVUkoJiIZGhcbGBQ8MCmGalusiXelh3e8m4nVrpjLpZGGbmEoIR0AAAALDAvCnoretaHmu6fNqpexmom7o5DPq5eLcmQrJCBRSUOTfnHYsZ2lj4NXU1BLSESPe3CdhnmDdm9waGB9cWtXUlewnI6pmZba0sy+trjmY4pzAAAAo3RSTlMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEJFRMVFw8KBj96m7u5s7Snh0YTGoLe/f7GMB+k+v23MznagAMOiMr74UkOP9b72mYFBWXd+vnckjAHBk29/cw5CQRw4/fSWwQCXuT2yGkIAU7g9uf6dgMPtdlt3JIMErfOH3voWQRbyo8JQ+yQBAIiMxM1s20EU1US4AAAAAFiS0dEEwy7XJYAAAAHdElNRQflARkNJSPYYCPyAAAAAW9yTlQBz6J3mgAAALFJREFUGBkFwTEuRUEABdB75s3LzESptgCR6P4aVCSs4CdUEjrLEIloBJ0KpYI1KOSXEhsQFa3GOYJNwAq6LQC8/dVpgSW4pXQ7LAGuebXrCACX7OMY4MJLfT5gAOca5rU9nHAJj6ZpfG7P8/uirOb56aP+1PyuNw7dNarv1GRqFA3TV1KT0ngwuPlKUpJy31prrbVpI0lNqgZUSUpST3vvvfd+VpKUpIwxxlXvvU9J/gHojhx4tq9qVQAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMS0wMS0yNVQxMzozNzozNSswMDowMOV/Fz8AAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjEtMDEtMjVUMTM6Mzc6MzUrMDA6MDCUIq+DAAAAAElFTkSuQmCC"></a>

~~-~~

**Code:**
```
<a href="https://github.com/EddieJaoudeCommunity"><img src="https://img.shields.io/static/v1?label=Guild&message=EddieHub&color=blueviolet&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAChVBMVEUPCwukzr0WFhWBhnYzMzAAAAAPEhUCAgc2LS4lJSQhIB8ODw4oKSgcGhsbHRsXFhUZGRk8My4hISD///+6nIyVcV5DODNsWE0aMFxASVdxW1EhO29JVWkLLnQTIUMAPJ5IPjISEhApMTYbAgBLT1FtZWQKEhFZAgIyDw9tAgKJDAtOBgcAAAAAAAAUFRMhIiEODQ0HBwcBAAEBAAAREBElJiQxMi4xMiwvMSswMiwnJyMeHxsYGRYVFRMbGhkKCAgfHh4eHxwsLSg2ODElJiIpKSc1NDQnJCMgIB4cHRoaGxgpKiktLS8VFBMSEhAgICAFAAAGBgUNDg5dSDyqh3MVFBQqKyoGCAqfgXJ5Z12tineEa1s7NjIAAAAAAAI2LyxhVUxzZVt8aF2cemmwh3KHaFg7MCuDYU+OfXOMf3eOf3VPRk50WlCBY1QrQmsHIlg8QVkmOmcLJlwpPWNGTEMwSHgOLGheaoQ+SGshNWgnQHJYW10AAJ8bNWtoa3STlJOKjZFHS1k6OEIAAB89QEY7OjonJyYmJiYxMTMgIB8FAwA4LS1APz8wMC4ZGBdEQ0M2ODciJiVFAABtCgtiJCRFJyYACAhPKytSMzRIJiYSAABYAAGGBASHBQRpAgFmBwZoCAhkBwc0BAY5PDU6PDY4OjMzNS8wMSsrLSgdHxsjIh0tKSMoJiE3MixiUkdoVUkoJiIZGhcbGBQ8MCmGalusiXelh3e8m4nVrpjLpZGGbmEoIR0AAAALDAvCnoretaHmu6fNqpexmom7o5DPq5eLcmQrJCBRSUOTfnHYsZ2lj4NXU1BLSESPe3CdhnmDdm9waGB9cWtXUlewnI6pmZba0sy+trjmY4pzAAAAo3RSTlMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEJFRMVFw8KBj96m7u5s7Snh0YTGoLe/f7GMB+k+v23MznagAMOiMr74UkOP9b72mYFBWXd+vnckjAHBk29/cw5CQRw4/fSWwQCXuT2yGkIAU7g9uf6dgMPtdlt3JIMErfOH3voWQRbyo8JQ+yQBAIiMxM1s20EU1US4AAAAAFiS0dEEwy7XJYAAAAHdElNRQflARkNJSPYYCPyAAAAAW9yTlQBz6J3mgAAALFJREFUGBkFwTEuRUEABdB75s3LzESptgCR6P4aVCSs4CdUEjrLEIloBJ0KpYI1KOSXEhsQFa3GOYJNwAq6LQC8/dVpgSW4pXQ7LAGuebXrCACX7OMY4MJLfT5gAOca5rU9nHAJj6ZpfG7P8/uirOb56aP+1PyuNw7dNarv1GRqFA3TV1KT0ngwuPlKUpJy31prrbVpI0lNqgZUSUpST3vvvfd+VpKUpIwxxlXvvU9J/gHojhx4tq9qVQAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMS0wMS0yNVQxMzozNzozNSswMDowMOV/Fz8AAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjEtMDEtMjVUMTM6Mzc6MzUrMDA6MDCUIq+DAAAAAElFTkSuQmCC"></a>
```

~~-~~

**Newbie? If you want to make it center, here's the code:**
```
<p align="center"><a href="https://github.com/EddieJaoudeCommunity"><img src="https://img.shields.io/static/v1?label=Guild&message=EddieHub&color=blueviolet&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAChVBMVEUPCwukzr0WFhWBhnYzMzAAAAAPEhUCAgc2LS4lJSQhIB8ODw4oKSgcGhsbHRsXFhUZGRk8My4hISD///+6nIyVcV5DODNsWE0aMFxASVdxW1EhO29JVWkLLnQTIUMAPJ5IPjISEhApMTYbAgBLT1FtZWQKEhFZAgIyDw9tAgKJDAtOBgcAAAAAAAAUFRMhIiEODQ0HBwcBAAEBAAAREBElJiQxMi4xMiwvMSswMiwnJyMeHxsYGRYVFRMbGhkKCAgfHh4eHxwsLSg2ODElJiIpKSc1NDQnJCMgIB4cHRoaGxgpKiktLS8VFBMSEhAgICAFAAAGBgUNDg5dSDyqh3MVFBQqKyoGCAqfgXJ5Z12tineEa1s7NjIAAAAAAAI2LyxhVUxzZVt8aF2cemmwh3KHaFg7MCuDYU+OfXOMf3eOf3VPRk50WlCBY1QrQmsHIlg8QVkmOmcLJlwpPWNGTEMwSHgOLGheaoQ+SGshNWgnQHJYW10AAJ8bNWtoa3STlJOKjZFHS1k6OEIAAB89QEY7OjonJyYmJiYxMTMgIB8FAwA4LS1APz8wMC4ZGBdEQ0M2ODciJiVFAABtCgtiJCRFJyYACAhPKytSMzRIJiYSAABYAAGGBASHBQRpAgFmBwZoCAhkBwc0BAY5PDU6PDY4OjMzNS8wMSsrLSgdHxsjIh0tKSMoJiE3MixiUkdoVUkoJiIZGhcbGBQ8MCmGalusiXelh3e8m4nVrpjLpZGGbmEoIR0AAAALDAvCnoretaHmu6fNqpexmom7o5DPq5eLcmQrJCBRSUOTfnHYsZ2lj4NXU1BLSESPe3CdhnmDdm9waGB9cWtXUlewnI6pmZba0sy+trjmY4pzAAAAo3RSTlMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEJFRMVFw8KBj96m7u5s7Snh0YTGoLe/f7GMB+k+v23MznagAMOiMr74UkOP9b72mYFBWXd+vnckjAHBk29/cw5CQRw4/fSWwQCXuT2yGkIAU7g9uf6dgMPtdlt3JIMErfOH3voWQRbyo8JQ+yQBAIiMxM1s20EU1US4AAAAAFiS0dEEwy7XJYAAAAHdElNRQflARkNJSPYYCPyAAAAAW9yTlQBz6J3mgAAALFJREFUGBkFwTEuRUEABdB75s3LzESptgCR6P4aVCSs4CdUEjrLEIloBJ0KpYI1KOSXEhsQFa3GOYJNwAq6LQC8/dVpgSW4pXQ7LAGuebXrCACX7OMY4MJLfT5gAOca5rU9nHAJj6ZpfG7P8/uirOb56aP+1PyuNw7dNarv1GRqFA3TV1KT0ngwuPlKUpJy31prrbVpI0lNqgZUSUpST3vvvfd+VpKUpIwxxlXvvU9J/gHojhx4tq9qVQAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMS0wMS0yNVQxMzozNzozNSswMDowMOV/Fz8AAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjEtMDEtMjVUMTM6Mzc6MzUrMDA6MDCUIq+DAAAAAElFTkSuQmCC"></a></p>
```

~~-~~

> We can customize the color

<a href="https://github.com/EddieJaoudeCommunity"><img src="https://img.shields.io/static/v1?label=Guild&message=EddieHub&color=black&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAChVBMVEUPCwukzr0WFhWBhnYzMzAAAAAPEhUCAgc2LS4lJSQhIB8ODw4oKSgcGhsbHRsXFhUZGRk8My4hISD///+6nIyVcV5DODNsWE0aMFxASVdxW1EhO29JVWkLLnQTIUMAPJ5IPjISEhApMTYbAgBLT1FtZWQKEhFZAgIyDw9tAgKJDAtOBgcAAAAAAAAUFRMhIiEODQ0HBwcBAAEBAAAREBElJiQxMi4xMiwvMSswMiwnJyMeHxsYGRYVFRMbGhkKCAgfHh4eHxwsLSg2ODElJiIpKSc1NDQnJCMgIB4cHRoaGxgpKiktLS8VFBMSEhAgICAFAAAGBgUNDg5dSDyqh3MVFBQqKyoGCAqfgXJ5Z12tineEa1s7NjIAAAAAAAI2LyxhVUxzZVt8aF2cemmwh3KHaFg7MCuDYU+OfXOMf3eOf3VPRk50WlCBY1QrQmsHIlg8QVkmOmcLJlwpPWNGTEMwSHgOLGheaoQ+SGshNWgnQHJYW10AAJ8bNWtoa3STlJOKjZFHS1k6OEIAAB89QEY7OjonJyYmJiYxMTMgIB8FAwA4LS1APz8wMC4ZGBdEQ0M2ODciJiVFAABtCgtiJCRFJyYACAhPKytSMzRIJiYSAABYAAGGBASHBQRpAgFmBwZoCAhkBwc0BAY5PDU6PDY4OjMzNS8wMSsrLSgdHxsjIh0tKSMoJiE3MixiUkdoVUkoJiIZGhcbGBQ8MCmGalusiXelh3e8m4nVrpjLpZGGbmEoIR0AAAALDAvCnoretaHmu6fNqpexmom7o5DPq5eLcmQrJCBRSUOTfnHYsZ2lj4NXU1BLSESPe3CdhnmDdm9waGB9cWtXUlewnI6pmZba0sy+trjmY4pzAAAAo3RSTlMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEJFRMVFw8KBj96m7u5s7Snh0YTGoLe/f7GMB+k+v23MznagAMOiMr74UkOP9b72mYFBWXd+vnckjAHBk29/cw5CQRw4/fSWwQCXuT2yGkIAU7g9uf6dgMPtdlt3JIMErfOH3voWQRbyo8JQ+yQBAIiMxM1s20EU1US4AAAAAFiS0dEEwy7XJYAAAAHdElNRQflARkNJSPYYCPyAAAAAW9yTlQBz6J3mgAAALFJREFUGBkFwTEuRUEABdB75s3LzESptgCR6P4aVCSs4CdUEjrLEIloBJ0KpYI1KOSXEhsQFa3GOYJNwAq6LQC8/dVpgSW4pXQ7LAGuebXrCACX7OMY4MJLfT5gAOca5rU9nHAJj6ZpfG7P8/uirOb56aP+1PyuNw7dNarv1GRqFA3TV1KT0ngwuPlKUpJy31prrbVpI0lNqgZUSUpST3vvvfd+VpKUpIwxxlXvvU9J/gHojhx4tq9qVQAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMS0wMS0yNVQxMzozNzozNSswMDowMOV/Fz8AAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjEtMDEtMjVUMTM6Mzc6MzUrMDA6MDCUIq+DAAAAAElFTkSuQmCC"></a> - <a href="https://github.com/EddieJaoudeCommunity"><img src="https://img.shields.io/static/v1?label=Guild&message=EddieHub&color=green&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAChVBMVEUPCwukzr0WFhWBhnYzMzAAAAAPEhUCAgc2LS4lJSQhIB8ODw4oKSgcGhsbHRsXFhUZGRk8My4hISD///+6nIyVcV5DODNsWE0aMFxASVdxW1EhO29JVWkLLnQTIUMAPJ5IPjISEhApMTYbAgBLT1FtZWQKEhFZAgIyDw9tAgKJDAtOBgcAAAAAAAAUFRMhIiEODQ0HBwcBAAEBAAAREBElJiQxMi4xMiwvMSswMiwnJyMeHxsYGRYVFRMbGhkKCAgfHh4eHxwsLSg2ODElJiIpKSc1NDQnJCMgIB4cHRoaGxgpKiktLS8VFBMSEhAgICAFAAAGBgUNDg5dSDyqh3MVFBQqKyoGCAqfgXJ5Z12tineEa1s7NjIAAAAAAAI2LyxhVUxzZVt8aF2cemmwh3KHaFg7MCuDYU+OfXOMf3eOf3VPRk50WlCBY1QrQmsHIlg8QVkmOmcLJlwpPWNGTEMwSHgOLGheaoQ+SGshNWgnQHJYW10AAJ8bNWtoa3STlJOKjZFHS1k6OEIAAB89QEY7OjonJyYmJiYxMTMgIB8FAwA4LS1APz8wMC4ZGBdEQ0M2ODciJiVFAABtCgtiJCRFJyYACAhPKytSMzRIJiYSAABYAAGGBASHBQRpAgFmBwZoCAhkBwc0BAY5PDU6PDY4OjMzNS8wMSsrLSgdHxsjIh0tKSMoJiE3MixiUkdoVUkoJiIZGhcbGBQ8MCmGalusiXelh3e8m4nVrpjLpZGGbmEoIR0AAAALDAvCnoretaHmu6fNqpexmom7o5DPq5eLcmQrJCBRSUOTfnHYsZ2lj4NXU1BLSESPe3CdhnmDdm9waGB9cWtXUlewnI6pmZba0sy+trjmY4pzAAAAo3RSTlMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEJFRMVFw8KBj96m7u5s7Snh0YTGoLe/f7GMB+k+v23MznagAMOiMr74UkOP9b72mYFBWXd+vnckjAHBk29/cw5CQRw4/fSWwQCXuT2yGkIAU7g9uf6dgMPtdlt3JIMErfOH3voWQRbyo8JQ+yQBAIiMxM1s20EU1US4AAAAAFiS0dEEwy7XJYAAAAHdElNRQflARkNJSPYYCPyAAAAAW9yTlQBz6J3mgAAALFJREFUGBkFwTEuRUEABdB75s3LzESptgCR6P4aVCSs4CdUEjrLEIloBJ0KpYI1KOSXEhsQFa3GOYJNwAq6LQC8/dVpgSW4pXQ7LAGuebXrCACX7OMY4MJLfT5gAOca5rU9nHAJj6ZpfG7P8/uirOb56aP+1PyuNw7dNarv1GRqFA3TV1KT0ngwuPlKUpJy31prrbVpI0lNqgZUSUpST3vvvfd+VpKUpIwxxlXvvU9J/gHojhx4tq9qVQAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMS0wMS0yNVQxMzozNzozNSswMDowMOV/Fz8AAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjEtMDEtMjVUMTM6Mzc6MzUrMDA6MDCUIq+DAAAAAElFTkSuQmCC"></a> - <a href="https://github.com/EddieJaoudeCommunity"><img src="https://img.shields.io/static/v1?label=Guild&message=EddieHub&color=blue&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAChVBMVEUPCwukzr0WFhWBhnYzMzAAAAAPEhUCAgc2LS4lJSQhIB8ODw4oKSgcGhsbHRsXFhUZGRk8My4hISD///+6nIyVcV5DODNsWE0aMFxASVdxW1EhO29JVWkLLnQTIUMAPJ5IPjISEhApMTYbAgBLT1FtZWQKEhFZAgIyDw9tAgKJDAtOBgcAAAAAAAAUFRMhIiEODQ0HBwcBAAEBAAAREBElJiQxMi4xMiwvMSswMiwnJyMeHxsYGRYVFRMbGhkKCAgfHh4eHxwsLSg2ODElJiIpKSc1NDQnJCMgIB4cHRoaGxgpKiktLS8VFBMSEhAgICAFAAAGBgUNDg5dSDyqh3MVFBQqKyoGCAqfgXJ5Z12tineEa1s7NjIAAAAAAAI2LyxhVUxzZVt8aF2cemmwh3KHaFg7MCuDYU+OfXOMf3eOf3VPRk50WlCBY1QrQmsHIlg8QVkmOmcLJlwpPWNGTEMwSHgOLGheaoQ+SGshNWgnQHJYW10AAJ8bNWtoa3STlJOKjZFHS1k6OEIAAB89QEY7OjonJyYmJiYxMTMgIB8FAwA4LS1APz8wMC4ZGBdEQ0M2ODciJiVFAABtCgtiJCRFJyYACAhPKytSMzRIJiYSAABYAAGGBASHBQRpAgFmBwZoCAhkBwc0BAY5PDU6PDY4OjMzNS8wMSsrLSgdHxsjIh0tKSMoJiE3MixiUkdoVUkoJiIZGhcbGBQ8MCmGalusiXelh3e8m4nVrpjLpZGGbmEoIR0AAAALDAvCnoretaHmu6fNqpexmom7o5DPq5eLcmQrJCBRSUOTfnHYsZ2lj4NXU1BLSESPe3CdhnmDdm9waGB9cWtXUlewnI6pmZba0sy+trjmY4pzAAAAo3RSTlMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEJFRMVFw8KBj96m7u5s7Snh0YTGoLe/f7GMB+k+v23MznagAMOiMr74UkOP9b72mYFBWXd+vnckjAHBk29/cw5CQRw4/fSWwQCXuT2yGkIAU7g9uf6dgMPtdlt3JIMErfOH3voWQRbyo8JQ+yQBAIiMxM1s20EU1US4AAAAAFiS0dEEwy7XJYAAAAHdElNRQflARkNJSPYYCPyAAAAAW9yTlQBz6J3mgAAALFJREFUGBkFwTEuRUEABdB75s3LzESptgCR6P4aVCSs4CdUEjrLEIloBJ0KpYI1KOSXEhsQFa3GOYJNwAq6LQC8/dVpgSW4pXQ7LAGuebXrCACX7OMY4MJLfT5gAOca5rU9nHAJj6ZpfG7P8/uirOb56aP+1PyuNw7dNarv1GRqFA3TV1KT0ngwuPlKUpJy31prrbVpI0lNqgZUSUpST3vvvfd+VpKUpIwxxlXvvU9J/gHojhx4tq9qVQAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAyMS0wMS0yNVQxMzozNzozNSswMDowMOV/Fz8AAAAldEVYdGRhdGU6bW9kaWZ5ADIwMjEtMDEtMjVUMTM6Mzc6MzUrMDA6MDCUIq+DAAAAAElFTkSuQmCC"></a> 

~~-~~

> Let's improve it first before we use it. Feel free to share your ideas

**Comment by Vyvy-vi** [comment]
We could maybe leave it up to the user to choose the color 🙃

**Comment by Vyvy-vi** [comment]
tho, maybe we could make an ico file for chibi and format it a little bit, so that it doesn't looked stretched

*chibi- referees to the Eddie emote

**Reply by vinzvinci** [reply]
Uhm let's try.

**Comment by stemount** [comment]
I actually quite like this, if the icon being stretched (and maybe needs to be switched for something that works in "icon like" sizes) problem is ironed out, it's a nice way to add the fact you're contributing to the EddieHub/community on your profile which could help more people discover the community but also be a badge to wear proud on your github profile.

Nice.

**Reply by vinzvinci** [reply]
Thanks @stemount , I will work for that.

**Comment by Akshu-on-github** [comment]
`EddieHub` could be on the left with the icon and the right... Could be something like `contributing` or `active` or `community`

We could also use that to indicate one of our roles in the server like `moderator`, `team eddie`, `contributor` etc

**Reply by vinzvinci** [reply]
Yes nice idea. How about this?

We have main badge **EddieHub** and badges for the roles with **EddieHub | Roles**

Can someone provide list of roles for the badges?

**Reply by Vyvy-vi** [reply]
We can do a daily contribution count on the badge, using github actions 🙃  🤔

**Reply by eddiejaoude** [reply]
Our community Action is collecting that info in the DB, so the API will be able to expose that 👍

**Reply by Vyvy-vi** [reply]
We can make a github action for this... 🤔 
Not sure about how we relate the API keys
